.run/Archipelago Unittests.run.xml

```diff
@@ -15,4 +15,4 @@
     <option name="_new_targetType" value="&quot;PATH&quot;" />
     <method v="2" />
   </configuration>
-</component>
\ No newline at end of file
+</component>
```

Comment:
Since this is an IDE-generated file, was this change done by pycharm or a different editor? If it's the latter, please undo.

Reply:
PyCharm was automatically changing the file and there were mentions of LF vs CRLF

I had other changes that were added for that file accidentally and reverting the changes still left the change here, leading me to believe that PyCharm is coercing the change.

Reply:
Modifying the file via the pycharm user interface does NOT add a line break at the end for me - if there is a line break, pycharm will remove it. Please revert this change.

Reply:
As a side note, the style guide does not apply to files that are automatically generated (e.g. by the IDE), and even then, XML is a bit special for two reasons:
- there are some tools/libs for xml that will not put a newline at end of file when saving.
- the final line of an xml file will always be the closing root tag and thus not be a merge conflict when "appending" to the file

Reply:
I had to edit the file outside of PyCharm since manually removing the newline in PyCharm would add it back when I saved the file. Change has been reverted though.